src/xpring-client.ts

```diff
@@ -18,11 +19,17 @@ class XpringClient {
    * The XpringClient will use gRPC to communicate with the given endpoint.
    *
    * @param grpcURL The URL of the gRPC instance to connect to.
+   * @param useNewProtocolBuffers If `true`, then the new protocol buffer implementation from rippled will be used. Defaults to false.
    */
-  public constructor(grpcURL: string) {
-    const defaultXpringClient = LegacyDefaultXpringClient.defaultXpringClientWithEndpoint(
+  public constructor(grpcURL: string, useNewProtocolBuffers: boolean = false) {
+    var defaultXpringClient: XpringClientDecorator = LegacyDefaultXpringClient.defaultXpringClientWithEndpoint(
```

Comment:
nit: Prefer `let` to `var`.

`let` got added to JS in 2015, and is block scoped (so if it's inside an if statement, it won't be valid outside the if block), whereas `var` is function scoped (which can lead to odd behavior if your mind isn't trained in the ways of JS).

In general, `let` is preferred by the community now.

Reply:
Acknowledged. Prettier rewrote this after I merged in #139 but I can also just use a ternary operator and make it a `const`.